java/sidecar/src/main/java/com/instaclustr/cassandra/sidecar/JMXHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.instaclustr.cassandra.sidecar;
+
+import javax.management.MBeanServerConnection;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+
+import java.io.IOException;
+
+import com.instaclustr.picocli.typeconverter.JMXServiceURLTypeConverter;
+
+public class JMXHelper {
```

Comment:
if this is purely a collection of helper/util functions, make the class `final` with a private constructor.